pyart/graph/radardisplay.py

```diff
@@ -122,21 +150,6 @@ def __init__(self, radar, shift=(0.0, 0.0)):
             warnings.warn('RadarDisplay does not correct for moving platforms')
         self.loc = (lat, lon)
```

Comment:
It is already being performed on a per sweep basis, self.x, self.y and self.z are not really being used (at least not inside this code)